src/runtime/threads/policies/hwloc_topology_info.cpp

```diff
@@ -1064,7 +1064,7 @@ namespace hpx { namespace threads
 
         {
             std::unique_lock<hpx::util::spinlock> lk(topo_mtx);
-            if (hwloc_get_thread_cpubind(topo, handle.native_handle(), cpuset,
+            if (hwloc_get_thread_cpubind(topo, (HANDLE)(handle.native_handle()), cpuset,
```

Comment:
Moreover, `HANDLE` is a Windows-only type, so this will break on other platforms.